docs/README.kernel.md

```diff
@@ -368,6 +399,13 @@ tar cz --exclude=./.git --exclude=$TEMP_MODULES . | ssh admin@$TARGET tar xz --n
 
 ### Using the Source with DKMS
 
+The following steps are all run on the target. This can be done either by
```

Comment:
Is there an 'or' that goes with the 'either' above? I suspect the 'or' is a 'serial console'?